cloud: Premium db auditing doc improvement (#23537)
pingcap/docs: tidb-cloud/premium/tidb-cloud-auditing-premium.md

> **Note:**
>
> If you disable log redaction, audit log files written to your cloud storage might contain sensitive information. This configuration is not recommended due to potential security risks.

## Specify audit filter rules

After enabling audit logging, you must specify auditing filter rules to control which user access events to capture and write to audit logs. If no filter rules are specified, TiDB Cloud does not log anything.

To specify auditing filter rules for an instance, take the following steps:

1. On the **DB Audit Logging** page, click **Add Filter Rule** in the **Log Filter Rules** section to add an audit filter rule.

2. In the **Add Filter Rule** dialog, configure the following items:

    - **Filter Name**: Enter a name for the filter rule.
    - **SQL User**: Enter the SQL user in the `<user>@<host>` format. The username and hostname can use `%` to match zero or more characters or `_` to match exactly one character. The `@` symbol and `<host>` are optional.
    - **Filter Events**: Select the events to log. For the supported filter events, see [Audit filter events](#audit-filter-events).

3. Click **Confirm** to add the filter rule.

> **Note:**
>

> > - Because audit logging consumes instance resources, be prudent when specifying filter rules. To minimize the consumption, it is recommended that you specify filter rules to limit the scope of audit logging to specific users and events, where possible.

Reviewer: ```suggestion
> - Because audit logging consumes instance resources, be prudent when specifying filter rules. To minimize resource usage, specify filter rules to limit audit logging to specific users and events where possible.
```

</CustomContent>

## Configure database audit logging settings

After configuring storage for your cloud provider, complete the step for database audit logging settings:

1. Set the log file rotation policy.

    You can rotate audit log files based on either file size or time interval. When either condition is met, TiDB Cloud generates a new audit log file.

2. Configure log redaction.

    Log redaction is enabled by default. When enabled, sensitive information in the SQL text is replaced with `?` in audit logs.

3. Click **Save and Enable** to apply the settings and enable audit logging.

> **Note:**
>
> If you disable log redaction, audit log files written to your cloud storage might contain sensitive information. This configuration is not recommended due to potential security risks.

> ## Specify auditing filter rules

Reviewer: ```suggestion
## Specify audit filter rules
```
